docs/general/emulator-support-and-issues.md

```diff
@@ -252,359 +207,227 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 - ✅ BizHawk core: **melonDS**
-- ❌ libretro core: **DeSmuME**
-- ❌ libretro core: **melonDS**
 
 ### Nokia N-Gage
 
-- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ _Not supported_.
 
 ### Oric
 
-- ❌ _Not supported_
+- ❌ _Not supported_.
 
 ### Palm OS
 
-- ❌ _Not supported_
-- ❌ libretro core: **Mu**
+- ❌ _Not supported_.
 
-### PC Engine | TurboGrafx-16 | SuperGrafx
+### PC Engine/TurboGrafx-16/SuperGrafx
 
 - ✅ libretro core: **Beetle PCE Fast**
-  - SuperGrafx games do not work on the Beetle PCE Fast core
+  - SuperGrafx games do not work on the Beetle PCE Fast core.
 - ✅ libretro core: **Beetle SuperGrafx**
-  - Most recommended
+  - Most recommended.
 - ✅ BizHawk core: **PCEHawk**
 
-### PC Engine CD | TurboGrafx-CD
+### PC Engine CD/TurboGrafx-CD
 
 - ✅ libretro core: **Beetle PCE Fast**
 - ✅ libretro core: **Beetle SuperGrafx**
-  - Most recommended
+  - Most recommended.
 - ✅ BizHawk core: **PCEHawk**
-- ❌ libretro core: **Beetle PCE**
-  - Does not expose RAM needed for PC Engine CD
 
 ### PC-6001
 
-- ❌ _Not supported_ - needs hashing method and memory map
+- ❌ _Not supported_.
 
-### PC-8800
+### PC-8000/8800
 
-- ✅ Standalone emulator: **[RAQuasi88](https://retroachievements.org/download.php#raquasi88)**
-  - Most recommended
-- ❌ libretro core: **quasi88**
-  - NOT RECOMMENDED
-  - PC-88VA not supported
-  - 16 Apr 2022 - Cannot load m3u. Hash fails because core locks disk file.
-  - 16 Apr 2022 - Attempting to load a single disk game in RALibretro just goes to "How many disks?" prompt. Seems to work in RetroArch
-  - 16 Apr 2022 - RALibretro does not provide subsystem interface for loading multi-disk games
+- ✅ Standalone emulator: **[RAQUASI88](https://retroachievements.org/download.php#raquasi88)**
 
 ### PC-9800
 
-- ❌ _Not supported_ - needs hashing method and memory map
-- ❌ libretro core: **Neko Project II Kai**
-  - Memory is not exposed.
-    - It appears to be exposed in some games depending on the RAM size selected, but it seems like the machine RAM and not actually related to the game.
-- ❌ libretro core: **Neko Project II**
+- ❌ _Not supported_.
 
 ### PC-FX
 
 - ✅ libretro core: **Beetle PC-FX**
 - ✅ BizHawk core: **T.S.T.** (Mednafen)
 
-### Philips CD-I
+### Philips CD-i
+
+- ❌ _Not supported_.
 
-- ❌ _Not supported_ - needs hashing method and memory map
-- ❌ libretro core: **CDi 2015**
-- ❌ libretro core: **SAME CDi**
+### Philips P2000T
 
-### Philips P2000
+- ❌ _Not supported_.
 
-- ❌ _Not supported_
-- ❌ libretro core: **M2000**
+### Pico
+
+- ❌ _Not supported_.
 
 ### Pico-8
 
-- ❌ _Not supported_ - needs hashing method and memory map
-- ❌ libretro core: **FAKE-08**
-- ❌ libretro core: **Retro8**
+- ❌ _Not supported_.
 
 ### PlayStation
 
 - ✅ Standalone emulator: **[DuckStation](https://www.duckstation.org/wiki/Main_Page)**
-  - There may be memory leak and/or burn-in when using save states. Softcore players beware!
+  - There may be memory leak and/or burn-in when using save states.
```

Comment:
While we're touching this line, may be worth fixing this typo:
```suggestion
  - There may be memory leaks and/or burn-in when using save states.
```

Reply:
Fixed. I didn't really notice this before, so it's either one of my accidents or it was already like that before my edits.